.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -32,6 +32,10 @@
 
 > /area driver-ebpf
 
+> /area driver-modern_ebpf
```

Comment:
```suggestion
> /area driver-modern-ebpf
```
Just personal style. Feel free to ignore my suggestion.

Reply:
Thank you! I removed the `e` from both `ebpf` drivers since we always refer to them with `bpf` only in our codebase. Moreover, the new engines will be probably called  `bpf` and `modern_bpf` so I think it is better to remove the `e`